<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bm::gui::FastGameOverDialog</class>
 <widget class="QDialog" name="bm::gui::FastGameOverDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="onceMoreGameButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Play again</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
